### ISSUE TITLE
Inviting to role and then collaborating maybe breaks things

### ISSUE THREAD

**Issue by gsteinLTU** [NetsBlox/NetsBlox#2427]
Someone said they had an issue with this at the camp.

**Comment by alexanderwebber** [comment]
@gsteinLTU could you elaborate on what you mean by "break"? Do you have any logs or project files?

**Comment by gsteinLTU** [comment]
@alexanderwebber No, sorry, it was pretty much a secondhand report. Some students were getting projects desynced from not understanding the exact collaboration flow.
